react-app/package.json

```diff
@@ -12,6 +12,9 @@
   "files": [
     "dist"
   ],
+  "engines": {
+    "node": ">=18.0"
```

Comment:
This shouldn't be necessary, and causes the gh-pages workflow to fail.

Reply:
Removed